src/e2e/java/teammates/e2e/pageobjects/AppPage.java

```diff
@@ -429,6 +429,10 @@ protected void markOptionAsUnselected(WebElement option) {
      */
     protected String getSelectedDropdownOptionText(WebElement dropdown) {
         Select select = new Select(dropdown);
+
+        //wait for dropdown text to load
+        this.waitForPageToLoad();
```

Comment:
It is overkill to use this method if just waiting for one specific item to load

Reply:
Yes, will look into a better way to wait for the text to be loaded